### DISCUSSION TITLE
View via query [Jun 02]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2311] [Discord Threads]
> 🙋‍♂️ **duongtu**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@867557959062913024> You can create views on backend directly using SQL queries & then trigger meta-sync to bring it over to NocoDB.   
####    
#### MetaSync is discussed here : https://docs.nocodb.com/developer-resources/webhooks  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/981457946577887292)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
